### PR TITLE
Fix crash without BuildCraft

### PR DESCRIPTION
**Pull request by miozune**
Close https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/issues/382

### CODE REVIEW

**Comment by firenoo** [comment]
Wondering if we even need this feature anymore, or if we want to extend it to all wrenches

**Comment by miozune** [comment]
Made it allow BC and/or AE wrench

File: src/main/java/appeng/parts/p2p/PartP2PTunnelNormal.java [hunk 1]
```diff
@@ -73,10 +72,12 @@ public boolean onPartActivate(final EntityPlayer player, final Vec3 pos) {
...
             mc.notifyUser(player, MemoryCardMessages.INVALID_MACHINE);
-        } else if (!player.isSneaking() && is != null && is.getItem() instanceof IToolWrench && !Platform.isClient()) {
-            printConnectionInfo(player);
-        } else if (tt != null) // attunement
-        {
+        } else if (!player.isSneaking()
+                && Platform.isWrench(player, is, (int) pos.xCoord, (int) pos.yCoord, (int) pos.zCoord)
+                && !Platform.isClient()) {
+                    printConnectionInfo(player);
+                    // spotless:off
```
Comment (firenoo) [inline src/main/java/appeng/parts/p2p/PartP2PTunnelNormal.java:79]: nit: alignment

Comment (miozune) [reply src/main/java/appeng/parts/p2p/PartP2PTunnelNormal.java:79]: spotless formats `// spotless:off` to here...

Comment (firenoo) [reply src/main/java/appeng/parts/p2p/PartP2PTunnelNormal.java:79]: ive been pretty upset with spotless lately

**Review by firenoo — APPROVED** [review]

**Pull request merged**